src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -180,31 +187,42 @@
     sendProductEvent(event, category, properties);
 
     if (COMPLETED_ORDER_PATTERN.matcher(event).matches()) {
+
+      // initialize instance of EventHitBuilder
+      EventHitBuilder eventHitBuilder = new EventHitBuilder();
+
       List<Product> products = properties.products();
       if (!isNullOrEmpty(products)) {
         for (int i = 0; i < products.size(); i++) {
           Product product = products.get(i);
-          ItemHitBuilder hitBuilder = new ItemHitBuilder();
-          hitBuilder.setTransactionId(properties.orderId())
+
+          // initialize instance of newProduct
+          com.google.android.gms.analytics.ecommerce.Product newProduct =
+                  new com.google.android.gms.analytics.ecommerce.Product()
+              .setId(product.id())
               .setName(product.name())
-              .setSku(product.sku())
               .setPrice(product.price())
-              .setQuantity(product.getLong(QUANTITY_KEY, 0))
-              .build();
-          attachCustomDimensionsAndMetrics(hitBuilder, properties);
-          Map<String, String> hit = hitBuilder.build();
-          tracker.send(hit);
-          logger.verbose("tracker.send(%s);", hit);
+              .setQuantity(product.getInt(QUANTITY_KEY, 0));
+
+          // add product to builder object
+          eventHitBuilder.addProduct(newProduct);
         }
       }
-      TransactionBuilder transactionBuilder = new TransactionBuilder();
-      transactionBuilder.setTransactionId(properties.orderId())
-          .setCurrencyCode(properties.currency())
-          .setRevenue(properties.total())
-          .setTax(properties.tax())
-          .setShipping(properties.shipping());
-
-      Map<String, String> transaction = transactionBuilder.build();
+
+      ProductAction productAction = new ProductAction(ProductAction.ACTION_PURCHASE)
+          .setTransactionId(properties.orderId())
+          .setTransactionRevenue(properties.total())
+          .setTransactionTax(properties.tax())
+          .setTransactionShipping(properties.shipping());
+
+      eventHitBuilder.setProductAction(productAction)
+             .setCategory(ENHANCED_ECOMMERCE);
```

Comment:
Shouldn't the category be what's passed in properties instead of a hardcoded string?
 https://developers.google.com/android/reference/com/google/android/gms/analytics/ecommerce/Product.html#setCategory(java.lang.String)